packages/lit/src/index.max.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import './polyfill-support.js';
+
+export * from './index.js';
+
+export * from './async-directive.js';
+export * from './decorators.js';
+export * from './directive-helpers.js';
+export * from './directive.js';
+export * from './directives/async-append.js';
+export * from './directives/async-replace.js';
+export * from './directives/cache.js';
+export * from './directives/choose.js';
+export * from './directives/class-map.js';
+export * from './directives/guard.js';
+export * from './directives/if-defined.js';
+export * from './directives/join.js';
+export * from './directives/keyed.js';
+export * from './directives/live.js';
+export * from './directives/map.js';
+export * from './directives/range.js';
+export * from './directives/ref.js';
+export * from './directives/repeat.js';
+export * from './directives/style-map.js';
+export * from './directives/template-content.js';
+export * from './directives/unsafe-html.js';
+export * from './directives/unsafe-svg.js';
+export * from './directives/until.js';
+export * from './directives/when.js';
```

Comment:
Maybe we should add a `directives.js` that groups all of these, like `decorators.js`?

Reply:
I don't think we should.

The rationale for the current setup is that there are more directives and they are less likely to be used in a known combination than the decorators. There were even fewer decorators at first and the core were pretty likely to be used together: `@customElement`, `@property`, and `@state` at least. Decorators also have to be used with a build chain, so they're more likely to be used with tree-shaking optimizers.

I'd rather move towards separating the decorators, but we can't do that now.